integration-tests/hazelcast-client/src/test/java/io/quarkus/it/hazelcast/client/HazelcastServerTestResource.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.it.hazelcast.client;
+
+import java.util.Collections;
+import java.util.Map;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class HazelcastServerTestResource implements QuarkusTestResourceLifecycleManager {
```

Comment:
:+1:

Reply:
Perhaps this could also be mentioned in the doc? Otherwise I think it will be unlikely most users will discover it.

Reply:
That's just a part of testing infrastructure under `src/test`, I believe that should not be exposed. If it is somehow, it's a bug, I believe

Reply:
Used only here: https://github.com/quarkusio/quarkus/blob/6b7fbe650cd3fee005ad6b94485f21491ac1053b/integration-tests/hazelcast-client/src/test/java/io/quarkus/it/hazelcast/client/HazelcastClientFunctionalityTest.java#L12

Reply:
Yeah, but aren't users supposed to be able to use this resource? I mean the other resources we have users can use and are encouraged to use actually

Reply:
I did not really realize it's going to be public, but well, no problem :) I will double-check and add a note in the docs

Reply:
See for example: `DerbyDatabaseTestResource` or `H2DatabaseTestResource`. I mean it's up to you if you want to expose it or not.
But wouldn't it be useful for users to have?

Reply:
Actually, it's quite useful :) yeah, would love to move forward with that

Reply:
Excellent!

Reply:
I exposed the functionality via `quarkus-test-hazelcast` :)

Reply:
With accompanying documentation, :+1: